pkg/config/config_params.go

```diff
@@ -106,10 +107,11 @@ type Config struct {
 	HealthHost    string `config:"string;localhost"`
 	HealthPort    int    `config:"int(0,65535);9098"`
 
-	PrometheusMetricsEnabled        bool `config:"bool;false"`
-	PrometheusMetricsPort           int  `config:"int(0,65535);9093"`
-	PrometheusGoMetricsEnabled      bool `config:"bool;true"`
-	PrometheusProcessMetricsEnabled bool `config:"bool;true"`
+	PrometheusMetricsEnabled        bool   `config:"bool;false"`
+	PrometheusMetricsHost           string `config:"prometheus-host;0.0.0.0"`
```

Comment:
I think we need to use an empty string here. I wasn't sure at first but after looking at https://stackoverflow.com/questions/49067160/what-is-the-difference-in-listening-on-0-0-0-080-and-80 I think it seems like we need empty instead of 0.0.0.0.

Reply:
Thanks, I have modified this.